doc/conf.py

```diff
@@ -1,74 +1,2618 @@
-# Configuration file for the Sphinx documentation builder.
```

Comment:
This file was erroneously overwritten with a `Doxyfile` -> should not be part of the repository.